testutils/helpers.go

```diff
@@ -42,13 +44,17 @@ func (f MockReporter) FrameMetadata(_ *reporter.FrameMetadataArgs) {}
 
 func StartTracer(ctx context.Context, t *testing.T, et tracertypes.IncludedTracers,
 	r reporter.SymbolReporter) (chan *host.Trace, *tracer.Tracer) {
+	// Without this intermediate variable, typecheck complains about float64/int conversion.
+	threshold := 0.01 * float64(math.MaxUint32)
+
 	trc, err := tracer.NewTracer(ctx, &tracer.Config{
 		Reporter:               r,
 		Intervals:              &MockIntervals{},
 		IncludeTracers:         et,
 		SamplesPerSecond:       20,
 		ProbabilisticInterval:  100,
 		ProbabilisticThreshold: 100,
+		OffCPUThreshold:        uint32(threshold),
```

Comment:
Can we use the following in here?
```
OffCPUThreshold: uint32(float64(0.01) * float64(math.MaxUint32)),
```

Similar to what is done in `internal/controller/controller.go`.

Reply:
Same error with golangci-lint v2.2.1 (latest version).